test/erlfmt_format_SUITE.erl

```diff
@@ -3232,6 +3260,22 @@ exportimport(Config) when is_list(Config) ->
         "    c/1,\n"
         "    c/2\n"
         "]).\n"
+    ),
+    ?assertSame(
+        "-export([\n"
+        "    bar/1, baz/1\n"
```

Comment:
Oh interesting.
I guess this is now a thing.
Almost feel like export should be special cased, but it is probably fine.

Reply:
Yes, I thought about this as well, but it seems consistent, if everything fits on a single line